tests/spi/get-file-content.go

```diff
@@ -67,28 +68,26 @@ var _ = framework.SPISuiteDescribe(Label("spi-suite", "get-file-content"), func(
 		})
 
 		It("SPIFileContentRequest should be in AwaitingTokenData phase", func() {
-			Eventually(func() bool {
+			Eventually(func() v1beta1.SPIFileContentRequestPhase {
 				SPIFcr, err = fw.AsKubeDeveloper.SPIController.GetSPIFileContentRequest(SPIFcr.Name, namespace)
 				Expect(err).NotTo(HaveOccurred())
 
-				return SPIFcr.Status.Phase == v1beta1.SPIFileContentRequestPhaseAwaitingTokenData
-			}, 2*time.Minute, 10*time.Second).Should(BeTrue(), "")
+				return SPIFcr.Status.Phase
+			}, 2*time.Minute, 10*time.Second).Should(Equal(v1beta1.SPIFileContentRequestPhaseAwaitingTokenData), fmt.Sprintf("SPIFileContentRequest %s/%s '.Status.Phase' field didn't have the expected value", SPIFcr.GetNamespace(), SPIFcr.GetName()))
```

Comment:
WDYT about also logging  the "actual value"? Something like:

```suggestion
                        }, 2*time.Minute, 10*time.Second).Should(Equal(v1beta1.SPIFileContentRequestPhaseAwaitingTokenData), fmt.Sprintf("SPIFileContentRequest %s/%s '.Status.Phase' field didn't have the expected value (expected: %s, actual: %s)", SPIFcr.GetNamespace(), SPIFcr.GetName(), v1beta1.SPIFileContentRequestPhaseAwaitingTokenData, SPIFcr.Status.Phase))
```

Reply:
Yes, you are right! There is no need for duplication! Tyyy